nyaa/views/users.py

```diff
@@ -100,6 +101,23 @@ def view_user(user_name):
         flask.flash(flask.Markup('User has been successfully {0}.'.format(action)), 'success')
         return flask.redirect(url)
 
+    if flask.request.method == 'POST' and ban_form and ban_form.nuke.data:
+        num_banned = 0
+        for t in chain(user.nyaa_torrents, user.sukebei_torrents):
+            t.deleted = True
+            t.banned = True
+            backend.tracker_api([t.info_hash], 'ban')
+            db.session.add(t)
+            num_banned += 1
+
+        log = "Nuked {0} torrents of {1}".format(num_banned, user.username)
+        adminlog = models.AdminLog(log=log, admin_id=flask.g.user.id)
```

Comment:
```py3
for log_class in (models.NyaaAdminLog, models.SukebeiAdminLog): # Check these because I'm writing this from memory
    adminlog = ...
    db.session.add(adminlog)
db.session.commit()
```
to add logs to both flavors. You could also count them separately (`Nuked x nyaa, y suke torrents`)